src/uniforms/MaterialsTexture.js

```diff
@@ -349,7 +349,7 @@ export class MaterialsTexture extends DataTexture {
 			// sample 13
 			index ++; // matte
 			index ++; // shadow
-			index ++;
+			floatArray[ index ++ ] = + m.vertexColors; // vertexColors
```

Comment:
nit: For better legibility I'd prefer to use `Number( m.vertexColors )` instead of `+ m.vertexColors`.